packages/odyssey-react/src/components/Modal/Modal.theme.ts

```diff
@@ -15,15 +15,23 @@ import type { ThemeReducer } from "@okta/odyssey-react-theme";
 export const theme: ThemeReducer = (theme) => ({
   AnimationDuration: theme.TransitionDurationBase,
   AnimationTimingFunction: theme.TransitionTimingBase,
+
   ContentFontSize: theme.FontSizeBody,
-  ContentPaddingBlockEnd: theme.SpaceScale5,
-  ContentPaddingBlockStart: theme.SpaceScale3,
+  ContentPaddingBlockEnd: theme.SpaceScale3,
+  ContentPaddingBlockStart: theme.SpaceScale1,
   ContentPaddingInline: 0,
+
   DialogBackgroundColor: theme.ColorBackgroundBase,
-  DialogPaddingBlockEnd: 0,
-  DialogPaddingBlockStart: theme.SpaceScale3,
+  DialogPaddingBlockEnd: theme.SpaceScale5,
+  DialogPaddingBlockStart: theme.SpaceScale5,
   DialogPaddingInline: theme.SpaceScale5,
+
   DismissMarginBlockEnd: theme.SpaceScale0,
-  FooterPaddingBlockEnd: theme.SpaceScale3,
+
+  FooterPaddingBlockStart: theme.SpaceScale3,
+  FooterPaddingBlockEnd: 0,
+
+  HeaderPaddingBlockEnd: theme.SpaceScale3,
```

Comment:
no start for header ?
HeaderPaddingBlockStart

Reply:
It's currently not in use. To keep breaking changes low, I'm trying to resist adding Tokens unless we're using them.